src/test/resources/application-test.yml

```diff
@@ -22,7 +22,7 @@ spring:
 
 hmpps:
   auth:
-    url: http://localhost:#WIREMOCK_PORT/auth
+    url: http://localhost:9090/auth
```

Comment:
Double-check if we still need wiremock before making this change official.

Reply:
Good shout. 
Our docker compose points at 9090 for auth. Will check with George